src/series/Pokemon.yml

```diff
@@ -75,12 +69,14 @@ games:
         platforms: []
         release_year: null
         sub-series: Core
+        comment: Gold/Silver, Crystal, Ruby / Sapphire, Emerald, FireRed / LeafGreen
     (III) Gen III all:
         genres: []
         platforms: []
         release_year: null
         sub-series: Core
-    (III) Gen III secondary:
+        comment: Pokémon Ruby Version / Pokémon Sapphire Version, Pokémon Emerald Version, Pokémon FireRed Version / Pokémon LeafGreen Version
+    (III) Pokémon Emerald:
```

Comment:
similar story here, I renamed `(III) Gen III secondary` to `(III) Pokémon Emerald:`

the added comment is for `(III) Gen III all`